csrankings-n.csv

```diff
@@ -170,6 +170,7 @@ Nathan S. Netanyahu,Bar-Ilan University,http://cs.biu.ac.il/en/node/644,xPv701UA
 Nathan Schneider 0001,Georgetown University,http://nathan.cl,3cozMf4AAAAJ
 Nathan Srebro,TTI Chicago,http://ttic.uchicago.edu/~nati,ZnT-QpMAAAAJ
 Nathan Sturtevant,University of Alberta,https://www.movingai.com,3utEUeoAAAAJ
+Nathan Weibe, University of Toronto,https://web.cs.toronto.edu/people/faculty-directory,DSgKHOQAAAAJ

```

Comment:
Web page needs to be faculty member's individual page.

Reply:
Nathan doesn't appear to have a webpage

https://cqiqc.physics.utoronto.ca/people/nathan-wiebe/
https://simons.berkeley.edu/people/nathan-wiebe

As is evident from the simons page, he uses a generic dept instead of his own!
https://phys.washington.edu/people/nathan-wiebe

Reply:
(SMH)
Also, there is a spurious space after the name; please remove. Thanks.

Reply:
I believe this is now fixed.